client/my-sites/site-settings/placeholder.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Main from 'components/main';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import SectionHeader from 'components/section-header';
+import SectionNav from 'components/section-nav';
+
+const Placeholder = ( { translate } ) => {
+	const header = (
+		<div className="site-settings__placeholder-item">
+			General Settings
```

Comment:
NIT- I find it strange to use "Example content" as the literal text of the placeholder blocks, although I believe that is the convention now.

In that line, should we use "Example section title" or an accurate section title, rather thant "General settings"?